Powdermills/Map/ParkMapView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ParkMapView.swift
+//  Powdermills
+//
+//  Created by James Sadlier on 09/11/2020.
+//
+
+import SwiftUI
+import Mapbox
+import Combine
+
+struct ParkMapView: View {
+	
+	/// View model of buildings list
+	@ObservedObject var buildingListViewModel: BuildingListViewModel
+	
+	/// View model for map view
+	@ObservedObject var viewModel = ParkMapViewModel()
+	
+	var body: some View {
+		VStack {
+			NavigationLink(
+				destination: buildingDetailsView() ,
+				isActive: $viewModel.showingDetails,
+				label: {
+					EmptyView()
+				})
+			MapView(viewModel: buildingListViewModel)
+				.centerCoordinate(ParkMapViewModel.parkCenterPoint).zoomLevel(16)
+				.edgesIgnoringSafeArea(.vertical)
+		}.onAppear {
+			viewModel.showingDetails = false
+		}
+	}
+	
+	func buildingDetailsView() -> AnyView {
+		guard let selecetdBuilding = buildingListViewModel.buildings.filter({$0.orders == viewModel.buildingOrders}).first else {
```

Comment:
would `buildingListViewModel.buildings.first(where: {$0.orders == viewModel.buildingOrders})` work?

Reply:
Ye it will, I'll swap it